terraform-provider-intro/order-create.md

```diff
@@ -0,0 +1,55 @@
+Now that you have initialized your Terraform workspace, you will create the order using Terraform.
+
+Add the following to your `main.tf`{{open}} file.
+
+<pre class="file" data-filename="main.tf" data-target="append">
```

Comment:
I think the initial `main.tf` could use an extra newline at the end of it, otherwise you end up with:

```
terraform {
  required_providers {
    hashicups = {
      versions = ["0.2"]
      source = "hashicorp.com/edu/hashicups"
    }
  }
}
provider "hashicups" {
  username = "education"
  password = "test123"
}
# ...
```

Which just looks a little ugly without a blank line between the blocks.